Барнаева М.К. Лаб. 1 Группа 6311

Barnaeva opened the pull request:
**ФИО:** Барнаева Марина
**Номер группы:** 6311  
**Номер лабораторной:** 1
**Краткое описание исходного кода:** Я взял(а) исходный код из интернета. Вот ссылка на него: [тут](https://github.com/Udinanon/telegram-facebook-bot/blob/master/FB_Bot.py)
**Краткое описание проделанной работы:** Я привела исходный код к стандарту PEP8.

NozdryakovaMarina commented on lab_2/fusing.py:
```diff
@@ -0,0 +1,49 @@
+        progress_bar_info[1].config(text=round(progress_bar_info[0]['value']))
+        progress_bar_info[2].update_idletasks()
+    pixel_matrix = a / b
+    for i in range(0, pixel_matrix.shape[0]):
+        for j in range(0, pixel_matrix.shape[1]):
+            flag_nan = True
+            if additional_channel[0][i][j] == 0:
+                pixel_matrix[i][j] = files[0][i][j]
+            if pixel_matrix[i][j] == 0:
+                pixel_matrix[i][j] = files[0][i][j]
+            if math.isnan(pixel_matrix[i][j]):
+                for m in range(0, len(files)):
+                    if flag_nan:
+                        flag_nan = False
+                        pixel_matrix[i][j] = 0
+                    pixel_matrix[i][j] += files[m][i][j] / len(files)
+            if pixel_matrix[i][j] < 0.0000005 or pixel_matrix[i][j] > 255:
+                pixel_matrix[i][j] = files[0][i][j]
+        progress_bar_info[0]['value'] += progress_step
+        progress_bar_info[1].config(text=round(progress_bar_info[0]['value']))
+        progress_bar_info[2].update_idletasks()
```
Довольно громоздкий блок кода. Возможно стоит разделить на несколько отдельных функций и попробовать оптимизировать их.

NozdryakovaMarina commented on lab_2/fusing.py:
```diff
@@ -0,0 +1,49 @@
+        progress_bar_info[0]['value'] += progress_step
+        progress_bar_info[1].config(text=round(progress_bar_info[0]['value']))
+        progress_bar_info[2].update_idletasks()
+    pixel_matrix = img_as_ubyte((pixel_matrix - np.min(pixel_matrix)) / (np.max(pixel_matrix) - np.min(pixel_matrix)))
```
Длинные вычисления лучше разбить на две строки.

NozdryakovaMarina commented on lab_2/fusing.py:
```diff
@@ -0,0 +1,49 @@
+    progress_bar_info[0]['value'] = 100
+    progress_bar_info[1].config(text=progress_bar_info[0]['value'])
+    progress_bar_info[2].update_idletasks()
+    return pixel_matrix
```
Пропущена пустая строка в конце модуля.

alxmcs reviewed (commented):
Зачтено

Barnaeva committed:
test

Barnaeva committed:
Fixed import order to comply with PEP 8 standards

Barnaeva committed:
Fixed E501 (too long line) according to PEP 8 recommendations.

Barnaeva committed:
F401: Unused imports removed
E302: Added blank lines between functions
E251: Fixed spacing around parameters

Barnaeva committed:
Fixed E128, E129, E131, E722 in accordance with PEP 8

Barnaeva committed:
Fixed F841 in accordance with PEP 8

Barnaeva committed:
Fixed N806,E123 in accordance with PEP 8

IlyaOv commented on lab_1/laba1.py:
```diff
@@ -0,0 +1,732 @@
+import argparse
+import configparser
+import csv
+import json
+import logging
+import re
+import time
+import cgi
+from urllib.parse import urlparse, parse_qs
+
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+from requests_html import HTMLSession
+
+from bs4 import BeautifulSoup
+
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
```
Импорты предлагаю поправить. Пустой строкой отделяются только группы импортов. Также хорошей практикой является расположение импортов в алфавитном порядке.

IlyaOv commented on lab_1/laba1.py:
```diff
@@ -0,0 +1,732 @@
+
+
+def get_mobile_URL(URL):
+    return URL[:8:] + "m" + URL[11::]
```
Предлагаю имена остальных переменных привести к стандарту PEP8.

Barnaeva committed:
N802, N803, N806 - Variable/function naming
E401, E402 - Import order and grouping

IlyaOv commented on lab_1/laba1.py:
```diff
@@ -0,0 +1,729 @@
+import time
+from urllib.parse import parse_qs, urlparse
+
+from bs4 import BeautifulSoup
```
Импорты from принято ставить в конце своей группы импортов.

IlyaOv reviewed (approved):
Зачтено
